test/e2e/handler/webhook_test.go

```diff
@@ -53,3 +54,24 @@ var _ = Describe("Mutating Admission Webhook", func() {
 		})
 	})
 })
+
+var _ = Describe("Validation Admission Webhook", func() {
+	Context("When a policy is created and progressing", func() {
+		BeforeEach(func() {
+			By("Creating a policy without waiting for it to be available")
+			updateDesiredState(linuxBrUp(bridge1))
+		})
+		AfterEach(func() {
+			waitForAvailablePolicy(TestPolicy)
+			updateDesiredStateAndWait(linuxBrAbsent(bridge1))
+			resetDesiredStateForNodes()
+		})
+		It("Should deny updating sequentially rolled out policy when it's in progress", func() {
+			By(fmt.Sprintf("Updating the policy %s", TestPolicy))
+			err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+				return setDesiredStateWithPolicyAndNodeSelector(TestPolicy, linuxBrUpNoPorts(bridge1), map[string]string{})
+			})
+			Expect(err).To(HaveOccurred())
```

Comment:
Let's match the error with the one we expect to be sure that we don't fail on retry limit.

Reply:
[force-pushed](https://github.com/nmstate/kubernetes-nmstate/compare/7ffdd64e52cb299be16bb65c8bb85a07585b338e..80f0cadf6ded6d8004cee6f9c6bd84334feafc19) change to use MatchError